docs/packaging.rst

```diff
@@ -2,5 +2,86 @@
 Packaging Guide
 ===============
 
-Documentation can be found at
-https://github.com/certbot/certbot/wiki/Packaging.
+Releases
+========
+
+We release packages and upload them to PyPI (wheels and source tarballs).
+
+- https://pypi.python.org/pypi/acme
+- https://pypi.python.org/pypi/certbot
+- https://pypi.python.org/pypi/certbot-apache
+- https://pypi.python.org/pypi/certbot-nginx
+
+The following scripts are used in the process:
+
+- https://github.com/letsencrypt/letsencrypt/blob/master/tools/release.sh
+
+We currently version with the following scheme:
+
+- ``0.1.0``
+- ``0.2.0dev`` for developement in ``master``
+- ``0.2.0`` (only temporarily in ``master``)
+- ...
+
+Notes for package maintainers
+=============================
+
+0. Please use our releases, not ``master``!
+
+1. Do not package ``certbot-compatibility-test`` or ``letshelp-certbot`` - it's only used internally.
+
+2. If you'd like to include automated renewal in your package ``certbot renew -q`` should be added to crontab.
+
+3. ``jws`` is an internal script for ``acme`` module and it doesn't have to be packaged - it's mostly for debugging: you can use it as ``echo foo | jws sign | jws verify``.
+
+4. Do get in touch with us. We are happy to make any changes that will make packaging easier. If you need to apply some patches don't do it downstream - make a PR here.
+
+Already ongoing efforts
+=======================
+
+
+Arch
+----
+
+From PyPI:
+- https://www.archlinux.org/packages/community/any/python2-acme
+- https://www.archlinux.org/packages/community/any/certbot
+- https://www.archlinux.org/packages/community/any/certbot-apache
+- https://www.archlinux.org/packages/community/any/certbot-nginx
+- https://www.archlinux.org/packages/community/any/letshelp-certbot
+
+From ``master``: https://aur.archlinux.org/packages/certbot-git
+
+Debian (and its derivatives, including Ubuntu)
+------
+
+https://alioth.debian.org/projects/letsencrypt/
+https://packages.debian.org/sid/certbot
+https://packages.debian.org/sid/python-certbot
+https://packages.debian.org/sid/python-certbot-apache
+
+Fedora
+------
+
+In Fedora 23+.
+
+- https://admin.fedoraproject.org/pkgdb/package/letsencrypt/
+- https://admin.fedoraproject.org/pkgdb/package/certbot/
+- https://admin.fedoraproject.org/pkgdb/package/python-acme/
+
+FreeBSD
+-------
+
+https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=203405
+
+GNU Guix
+--------
+
+- https://www.gnu.org/software/guix/package-list.html#letsencrypt
+
+OpenBSD
+-------
+
+- http://cvsweb.openbsd.org/cgi-bin/cvsweb/ports/security/letsencrypt/
+- https://github.com/letsencrypt/letsencrypt/pull/1175
+- https://github.com/letsencrypt/letsencrypt/issues/1174
```

Comment:
I think these two GitHub links should be removed. The issue links to a problem about OpenBSD support (which was solved by the creation of their package) and the PR was about adding OpenBSD support to `certbot-auto` which we never accepted because they have an official port.